bedevere/bpo.py

```diff
@@ -111,6 +111,20 @@ def create_success_status(found_issue):
                               target_url=url)
 
 
+def create_failure_status(issue_not_found_on_bpo, found_issue=None):
```

Comment:
There's no need to overload creating a failure case for the missing issue number in the PR versus the issue not existing. Just make them separate functions.